### PR TITLE
Symfonize stats position

### PR DESCRIPTION
**Pull request by jeromecombes**
Replace PR #227 
this PR include adaptation for teleworking #352

### CODE REVIEW

File: public/setup/atomicupdate/symfonize_stats_position.php [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?php
+
+$sql[]="DELETE` FROM {$dbprefix}acces` WHERE `page`='statistiques/postes.php' ;";
+
```
Comment (jeromecombes) [inline public/setup/atomicupdate/symfonize_stats_position.php:4]: revert quote misplaced
<pre>$sql[]="DELETE FROM `{$dbprefix}acces` WHERE `page`='statistiques/postes.php' ;";</pre>

File: templates/statistics/position.html.twig [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+                </td>
+              </tr>
+              {% if nbSites>1 %}
+                <tr style='vertical-align:top'>
+                  <td><label class='intitule'>Sites</label></td>
+                  <td>
+                    <select name='selectedSites[]' multiple='multiple' size='{{ nbSites+1 }}' onchange='verif_select("selectedSites");' class='ui-widget-content ui-corner-all' >
+                      <option value='Tous'>Tous</option>
+                        {% for i in 1..nbSites %}
+                          {% if i in selectedSites %}
```
Comment (jeromecombes) [inline templates/statistics/position.html.twig:68]: Site names are shifted in the dropdown menu.
- There is a blank option linked to the site 1
- The second option with the name of site 1 is linked to site 2
- ...

**Review by jeromecombes — COMMENTED** [review]
- with APP_DEBUG=1, I get "access denied" on /statistics/position
- on load, JS Error in console : document.form is undefined /statistics/position:16
- Delete button reload the page but does not reset selected options
- CSV and Excel links do not work, JS error: <pre>Uncaught TypeError: "#baseURL".val is not a function
    export_stat http://plb/js/script.js?version=20.11.00.014:794
    <anonymous> javascript:export_stat('postes','xls');:1
script.js:794:30</pre>

File: public/setup/atomicupdate/symfonize_stats_position.php (hunk 1 above)
Comment (jeromecombes) [reply public/setup/atomicupdate/symfonize_stats_position.php:4]: solved with this commit 1b606b802939cc354ec73244cee8708f259c99ff

File: templates/statistics/position.html.twig (hunk 2 above)
Comment (jeromecombes) [reply templates/statistics/position.html.twig:68]: solved with this commit 1b606b802939cc354ec73244cee8708f259c99ff

**Comment by jeromecombes** [comment]
>     * with APP_DEBUG=1, I get "access denied" on /statistics/position
> 
>     * on load, JS Error in console : document.form is undefined /statistics/position:16
> 
>     * Delete button reload the page but does not reset selected options
> 
>     * CSV and Excel links do not work, JS error: Uncaught TypeError: "#baseURL".val is not a function
>       export_stat http://plb/js/script.js?version=20.11.00.014:794
>        javascript:export_stat('postes','xls');:1
>       script.js:794:30


This commit 1b606b802939cc354ec73244cee8708f259c99ff fix CSV and Excel links

**Comment by alexarn** [comment]
Pull request rebased on master.

+ Apply the following changes since this PR has been submitted:
  - MT31714 : replace the join function with the implode function,
  - Fix HTML entities in floors and groups dropdown menus (position's form) 
  - New Planno theme

**Comment by jeromecombes** [comment]
TEST PLAN : 
Tester les statistiques par poste
comparer avec une instance master, avec les mêmes données : les résultats doivent être identiques.
tester en mono et multi-sites
tester avec de nombreuses données (jeu de données data1 ou data2 en regardant les stats sur les années chargées (2017, 2018 ou 2019)

**Comment by alexarn** [comment]
Pull request rebased on master

**Review by jeromecombes — COMMENTED** [review]
Note : le bouton "Effacer" ne fait rien (reload de la page mais les valeurs restent les mêmes). Le comportement est identique sur master, donc je passe.

PR 357 approuvée pour master

**Comment by alexarn** [comment]
Merged on master.
DBRev 22.05.00.002

**Pull request closed without merging**